### PR TITLE
Make List View Responsive

### PR DESCRIPTION
**Pull request by SMahil**
Make List View Responsive

### CODE REVIEW

File: src/app/app.component.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import { Component } from '@angular/core';
+import { AlmDropdown, AlmDropdownToggle } from './shared-component/alm-dropdown/alm-dropdown';
```
Comment (sanbornsen) [inline src/app/app.component.ts]: - Where are these files?
- We are using **pf** instead of **alm** for any independent component's prefix.

**Comment by sanbornsen** [comment]
@SMahil can you get the errors fixed?

**Comment by nimishamukherjee** [comment]
PR shows errors : 
error

**Comment by joshuawilson** [comment]
Both unit and functional tests fail.

**Pull request merged**